### PR TITLE
completed box model

### PR DESCRIPTION
**Pull request by bug-smith**
Completed.
266095083-5d892db1-e113-4760-bf10-1520a68426ea

### CODE REVIEW

File: css-box-model/styles.css [hunk 1]
```diff
@@ -0,0 +1,12 @@
+button.language {
+  font-size: 24px;
+  background-color: lightgray;
+
+  /* your code here 👇 */
+  box-sizing: border-box;
+  margin-left: 12px;
+  margin-right: 16px;
```
Comment (shawnkost) [inline css-box-model/styles.css]: These should be the same value

File: css-box-model/styles.css [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  padding-top: 6px;
+  padding-bottom: 6px;
```
Comment (shawnkost) [inline css-box-model/styles.css:10]: Missing a `padding-right` & `padding-left` of `16px`

**Review by shawnkost — CHANGES_REQUESTED** [review]
Couple changes here Bug!

**Comment by bug-smith** [comment]
Screenshot 2023-09-08 at 9 57 37 AM
revisions completed.

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**